scanner/src/main/kotlin/scanners/ScanCode.kt

```diff
@@ -397,33 +398,49 @@ class ScanCode(config: ScannerConfiguration) : LocalScanner(config) {
     }
 
     /**
-     * Return the copyright statements in the vicinity, as specified by [toleranceLines], of [startLine]. The default
-     * value of [toleranceLines] is set to 5 which seems to be a good balance between associating findings separated by
-     * blank lines but not skipping complete license statements.
+     * Return the copyright statements in the vicinity, as specified by [toleranceLines], of [licenseStartLine]. The
+     * default value of [toleranceLines] is set to 5 which seems to be a good balance between associating findings
+     * separated by blank lines but not skipping complete license statements.
```

Comment:
Please also update the comment mentioning what the path parameter is about.

Reply:
Done.